build.assets/Dockerfile

```diff
@@ -153,7 +158,7 @@ RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --pr
     rustup --version && \
     cargo --version && \
     rustc --version && \
-    rustup component add --toolchain $RUST_VERSION-x86_64-unknown-linux-gnu rustfmt clippy && \
+    rustup component add rustfmt clippy && \
```

Comment:
Since official builds will (for now) continue to run on amd64, you can probably further simplify this by only adding the cross compilation targets on amd64 and just using the default rustup target otherwise.

Reply:
I think there is one more reason why removing other build targets makes sense. This image doesn't contain `gcc-multilib` package on arm, so cross-compilation won't work anyways. I think that there is a way to install that package if we really need it, but for now I think that not installing the it is fine.

Reply:
Agreed, lets remove them!